### ISSUE TITLE
Cannot Open Tests From Results Window if VI Left Memory and is in Nested LVLIB

### ISSUE THREAD

**Issue by jacobson3** [JKISoftware/Caraya#47]
When trying to double click a failing assert to open up the BD of the failing test, you will receive error 7 if the VI has left memory and is inside of a nested lvlib, e.g. Unit Tests.lvlib:FPGA Tests.lvlib:FPGA Test Suite.vi. This is due to the match pattern function only looking for the text coming after ".lvlib:" which results in Caraya trying to find the file "FPGA Tests.lvlib:FPGA Test Suite.vi" in example given. For my simple use case I was able to resolve this by modifying "Caraya.lvlib:Basic Test Manager.lvclass: Open Block Diagram.vi" as follows.

![Open Block Diagram](https://user-images.githubusercontent.com/25009788/67881639-d9944e80-fb0e-11e9-8a04-39804c4c8a5d.png)

Basically, just change the match pattern primitive to match regular expression with the regular expression " \.lvlib:(?!.*\.lvlib) ". This regular expression should perform the same search for ".lvlib:" but only if another ".lvlib:" does not come afterwords.

I can make this edit and pull request myself if someone can tell me what version of LV I should be using.

**Comment by francois-normandin** [comment]
Hey Matthew, 
good catch! The project is in LV2013 if you want to make those changes.

**Comment by francois-normandin** [comment]
fixed in 1.0 release branch
